src/Apps/__tests__/Fixtures/Auction/Routes/Register/index.ts

```diff
@@ -1,23 +1,44 @@
-import { redirects_me } from "__generated__/redirects_me.graphql"
-import { redirects_sale } from "__generated__/redirects_sale.graphql"
+import { Register_me } from "__generated__/Register_me.graphql"
+import { Register_sale } from "__generated__/Register_sale.graphql"
+import { routes_RegisterQueryResponse } from "__generated__/routes_RegisterQuery.graphql"
 
-export interface RegisterQueryResponse {
-  sale: redirects_sale
-  me: redirects_me
+type Omit<T, K> = Pick<T, Exclude<keyof T, K>>
```

Comment:
Omit is a first class member of TS now, you shouldn't need to redefine it. 

https://www.typescriptlang.org/docs/handbook/utility-types.html#omittk

Reply:
I'm pretty sure we do need it unless we bump the ts version; it was working fine because we hadn't set pinned it to the editor version in `code`, but then it failed in type-checking

Reply:
All right, we can leave it for now. I'll look into bumping the version.